SAGE Journals.js

```diff
@@ -168,6 +168,10 @@ function scrape(doc, url) {
 					}
 				}
 			}
+			//ubtue: add tag "Book Review" in every issue 5 of specific journals if the dc.Type is "others"
+			let reviewType = ZU.xpathText(doc, '//meta[@name="dc.Type"]/@content');
+			if (reviewType && reviewType.match(/other/i) && item.issue === '5' && item.ISSN === '0142-064X' || item.ISSN === '0309-0892') item.tags.push('Book Review') && item.notes.push({note: "Booklist:" + item.date.match(/\d{4}$/)});
```

Comment:
For readability and performance please check for the relavant ISSNs first:  `if (item.ISSN === ...) { ... }`.

Reply:
ok